package.json

```diff
@@ -27,6 +27,18 @@
     "eslint-config-godaddy": "^4.0.1"
   },
   "bin": {
-    "orglinter": "./src/bin/cli.js"
-  }
+    "orglinter": "./bin/cli.js"
+  },
+  "keywords": [
+    "github",
+    "linting",
+    "organization-management"
+  ],
```

Comment:
Do you want to have an [`engines`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines) property to indicate node versions? I don't feel too strong about it and have no real preference as long as you're not using anything that isn't in the oldest node LTS version.

Reply:
Good call! I think it'd be preferable to pin this to modern-ish versions. Do you have a recommendation on what that property might contain?

Again referencing `@gasket/cli`, I see:
```json
  "engines": {
    "node": ">= 8.0.0"
  },
```
Does that seem reasonable? Or should we maybe look even more modern than that?

Reply:
[LTS schedule](https://nodejs.org/en/about/releases/)

node@8 is already out of LTS completely, so we should definitely be more modern than that.
node@10 will be EOL'd very soon (2021-04-30)

I'd suggest node>=12 at a minimum